src/utils/__tests__/isKnownUnit-test.js

```diff
@@ -32,5 +33,20 @@ test("isKnownUnit", t => {
   t.ok(isKnownUnit("dpi"))
   t.ok(isKnownUnit("dpcm"))
   t.ok(isKnownUnit("dppx"))
+
+  t.ok(isKnownUnit("%", { only: "length" }))
```

Comment:
Confused. Should these tests be negative, because they are *not* known *length* units? And shouldn't the `px` test be positive, because it *is* a known length unit?